- floor: 7
  altitude: 28
  map_name: '/eng2/7f'
- floor: 3
  altitude: 9
  map_name: '/eng2/3f'
- floor: 2
  altitude: 4.5
  map_name: '/eng2/2f'
- floor: 1
  altitude: 0
  map_name: '/eng2/1f'
